tools/internal_ci/linux/gcr_upload.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python2.7
```

Comment:
probably not the right directory. tools/internal_ci is scripts and configs to run on an internal CI platform (kokoro).